pkg/config/app/kuma-cp/config.go

```diff
@@ -464,6 +464,8 @@ type ExperimentalConfig struct {
 	// If true skips persisted VIPs. Change to true only if generateMeshServices is enabled.
 	// Do not enable on production.
 	SkipPersistedVIPs bool `json:"skipPersistedVIPs" envconfig:"KUMA_EXPERIMENTAL_SKIP_PERSISTED_VIPS"`
+	// If true uses Delta xDS to deliver changes to sidecars.
+	UseDeltaXds bool `json:"useDeltaXds" envconfig:"KUMA_EXPERIMENTAL_USE_DELTA_XDS"`
```

Comment:
```suggestion
	DeltaXds bool `json:"deltaXds" envconfig:"KUMA_EXPERIMENTAL_DELTA_XDS"`
```

It's simpler and just as expresive